spikeinterface/sorters/external/combinato.py

```diff
@@ -153,7 +153,7 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
         tmpdir = sorter_output_folder
 
         if verbose:
-            print(f'Running combinato in {tmpdir}...')
+            print(f'Running combinato in {tmpdir}..')
```

Comment:
What's up with all the cahnges from `...` to `..` here.

Reply:
Hi think I did a find and replace for relative paths (`...` -> `..`)! Will fix it